src/applications/pensions/config/form.js

```diff
@@ -1,6 +1,5 @@
 import merge from 'lodash/merge';
 import get from 'platform/utilities/data/get';
```

Comment:
Importing from platform via platform/utilities is deprecated. Import from @department-of-veterans-affairs/platform-utilities